scripts/autotools/libmongoc/CheckICU.m4

```diff
@@ -0,0 +1,40 @@
+PHP_ARG_WITH([mongodb-icu],
+             [whether to enable ICU for SASLPrep with SCRAM-SHA-256 authentication],
+             [AS_HELP_STRING([--with-mongodb-icu=@<:@auto/yes/no@:>@],
+                             [MongoDB: Enable ICU for SASLPrep with SCRAM-SHA-256 authentication [default=auto]])],
+             [auto],
+             [no])
+
+AS_IF([test "$PHP_MONGODB_ICU" = "auto" -o "$PHP_MONGODB_ICU" = "yes"],[
+  found_icu=no
+
+  PKG_CHECK_MODULES([PHP_MONGODB_ICU],[icu-uc],[
+    PHP_MONGODB_BUNDLED_CFLAGS="$PHP_MONGODB_BUNDLED_CFLAGS $PHP_MONGODB_ICU_CFLAGS"
+    PHP_EVAL_LIBLINE([$PHP_MONGODB_ICU_LIBS],[MONGODB_SHARED_LIBADD])
+    PHP_MONGODB_ICU="yes"
+	found_icu=yes
```

Comment:
This should have a four-space indent. Looks like you're using tabs.

Reply:
Fixed